src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -5466,9 +5466,7 @@ class spell_gen_shadowmeld : public AuraScript {
 
         if (AuraEffect const* aurEff = caster->GetAuraEffect(SPELL_RACIAL_ELUSIVENESS, EFFECT_0)) {
```

Comment:
remove the brackets. Also we don't inline brackets here, they always go into a new line. Brackets are only used for more than one action inside of conditions.